backend/typescript/rest/adminRoutes.ts

```diff
@@ -0,0 +1,30 @@
+import { Router } from "express";
+
+import AdminService from "../services/implementations/adminService";
+import IAdminService from "../services/interfaces/adminService";
+import { getErrorMessage } from "../utilities/errorUtils";
+import { waiverUpdateValidator } from "../middlewares/validators/adminValidators";
+
+const adminRouter: Router = Router();
+const adminService: IAdminService = new AdminService();
+adminRouter.post("/waiver", waiverUpdateValidator, async (req, res) => {
+  try {
+    const waiver = await adminService.updateWaiver({
+      clauses: req.body,
```

Comment:
should this be req.body.clauses? it's causing an error on the backend for me if I send an object like 
```
{
    "clauses": [
        {
            "text": "hello",
            "required": true
        }
    ]
}
```